src/main/java/stsjorbsmod/patches/ClarifyPatch.java

```diff
@@ -0,0 +1,37 @@
+package stsjorbsmod.patches;
+
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.screens.DrawPileViewScreen;
+import javassist.CannotCompileException;
+import javassist.expr.ExprEditor;
+import javassist.expr.MethodCall;
+import stsjorbsmod.powers.ClarifyPower;
+
+public class ClarifyPatch {
+    public static final String ClarifyPatchName = ClarifyPatch.class.getName();
+
+    @SpirePatch(clz = DrawPileViewScreen.class, method = "open")
+    @SpirePatch(clz = DrawPileViewScreen.class, method = "render")
+    public static class AbstractRoomPatch {
+        public static ExprEditor Instrument() {
+            final int[] counter = {0};
+            return new ExprEditor() {
+                @Override
+                public void edit(MethodCall methodCall) throws CannotCompileException {
+                    if (methodCall.getMethodName().equals("hasRelic")) {
+                        if (counter[0] == 0) {
+                            methodCall.replace(
+                                    "$_= $proceed($$) || "+ClarifyPatchName+".playerHasPower();"
```

Comment:
Today, Frozen Eye is the only relic the base game is calling `hasRelic` for in these methods, but it would probably be a good idea to make the patch a little more specific and only apply specifically to `hasRelic("Frozen Eye")` calls in case a different check gets added in the future (or by a patch from a different mod).

You can do that using the same pattern that `PlayableCursesPatch` uses, something like:

```java
methodCall.replace(String.format(
        "{ $_ = ($proceed($$) || (%1$s.isFrozenEye($1) && %1$s.hasClarifyPower($0))); }",
        ClarifyPatch.class.getName()));
```